README.md

```diff
@@ -29,6 +29,8 @@ jobs:
           merge_method: squash
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # Set your time zone. Default is UTC
+          TIME_ZONE: 'UTC'
```

Comment:
could you set it as `with` parameter instead? You will need to update [inputs in `action.yml`](https://github.com/gr2m/merge-schedule-action/blob/e76675203a2b4eef81ae5be434b4614ab1afbeb9/action.yml#L9-L15)

Reply:
Thank you your review.
I corrected them.

6d3a429